apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -50,6 +50,12 @@ type ClusterLogForwarderSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Forwarder Pipelines",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:forwarderPipelines"}
 	Pipelines []PipelineSpec `json:"pipelines,omitempty"`
 
+	// ServiceAccountName is the service account's name provided by the admin.
+	//
+	// +kubebuilder:validation:minLength:=1
```

Comment:
We need to remove this validation to support the legacy case.  If not in this PR already, we should add our own validation for the non-legacy case to require the SA.. maybe even validate it exists

Reply:
I do validate that the SA exists for the non-legacy case. I removed this validation